command/agent/agent.go

```diff
@@ -1093,6 +1093,11 @@ func (a *Agent) ShouldReload(newConfig *Config) (agent, http bool) {
 		agent = true
 	}
 
+	// Allow the ability to only reload HTTP connections
```

Comment:
Ha, have we just been copying and pasting this comment around or does it actually have a meaning that alludes me? :sweat_smile: 

I think we can just remove it. These comparisons aren't complex.